service.py

```diff
@@ -23,14 +23,12 @@ def signal_handler(sig: int, _frame):
     signal.signal(signal.SIGTERM, signal_handler)
     signal.signal(signal.SIGINT, signal_handler)
 
-    database = db.FAFDatabase(loop)
-    await database.connect(
+    database = db.FAFDatabase(
         host=config.DB_SERVER,
         port=int(config.DB_PORT),
         user=config.DB_LOGIN,
         password=config.DB_PASSWORD,
-        maxsize=10,
-        db=config.DB_NAME,
+        db=config.DB_NAME
```

Comment:
BTW it is a good idea to leave the trailing comma even when it's not strictly required as this will reduce the size of git diffs created when you modify the code. As you can see here, you removed one item yet generated 2 lines of git diffs because the trailing comma was also removed. Usually the way it happens is when you add an item to the end, you also have to add a trailing comma to the previous line which causes you to modify 2 lines rather than 1. If you always include the comma, then you only have to modify 1 line.
```suggestion
        db=config.DB_NAME,
```